### DISCUSSION TITLE
[Feature Request]: new column type: `ContextMenu`

### DISCUSSION THREAD

**Discussion by amshehzad** [rappasoft/laravel-livewire-tables#1484] [Feature Requests]
### Overview

It would be great to have a contextMenue option, just like we have clickable rows for example, or ButtonGroup column

**Comment by lrljoe** [comment]
https://discord.com/channels/1118369537155399690/1164021143066390548

**Comment by lrljoe** [comment]
@amshehzad is initially assigned, please advise if this is no longer the case.,
